app/src/main/java/org/fossasia/pslab/activity/MultimeterActivity.java

```diff
@@ -161,7 +161,7 @@ public void onClick(View view) {
                         capacitanceUnit = "nF";
                     } else if (capacitance < 1e-3) {
                         Capacitance = capacitanceFormat.format((capacitance / 1e-6));
-                        capacitanceUnit = "\u00B5"+"F";
+                        capacitanceUnit = "\u00B5" + "F";
                     } else if (capacitance < 1e-1) {
                         Capacitance = capacitanceFormat.format((capacitance / 1e-3));
                         capacitanceUnit = "mF";
```

Comment:
@yatri1609 These changes were done in my PR as said by @harsh-2711 please sync your branch with development branch of upstream try `git reset --soft upstream/development` after you have fetched the upstream.